src/Compilers/CSharp/Test/Emit2/Semantics/PrimaryConstructorTests.cs

```diff
@@ -19777,6 +19777,129 @@ class C(int p)
             AssertEx.Equal("System.Int32 C.this[System.Int32 i] { get; }", info.Symbol.ToTestDisplayString());
         }
 
+        [WorkItem("https://github.com/dotnet/roslyn/issues/75002")]
+        [Fact]
+        public void PartialMembers_01()
+        {
+            var source1 = """
+                C c = null;
+                c.M();
+                _ = c.P;
+                """;
+            var source2 = """
+                partial class C(int p)
+                {
+                    public partial void M() { }
+                    public partial void M();
+                    public partial object P { get; }
+                    public partial object P { get => null; }
+                }
+                """;
+            var comp = CreateCompilation([source1, source2]);
+            var tree = comp.SyntaxTrees[0];
+            var model = comp.GetSemanticModel(tree);
+            model.GetDiagnostics().Verify(
+                // (2,3): error CS0121: The call is ambiguous between the following methods or properties: 'C.M()' and 'C.M()'
+                // c.M();
+                Diagnostic(ErrorCode.ERR_AmbigCall, "M").WithArguments("C.M()", "C.M()").WithLocation(2, 3),
+                // (3,7): error CS0229: Ambiguity between 'C.P' and 'C.P'
+                // _ = c.P;
+                Diagnostic(ErrorCode.ERR_AmbigMember, "P").WithArguments("C.P", "C.P").WithLocation(3, 7));
+        }
+
+        [Fact]
+        public void NullableAttributes_01()
+        {
+            var source1 = """
+                #nullable enable
+                C c = null!;
+                object o;
+                o = c.M();
```

Comment:
I expected a nullable warning on this assignment and the one below, since the `[MaybeNull]` on the members causes the RHS to be maybe-null. Did I misunderstand something? If this test is demonstrating a bug it would be good to indicate that with a comment or an issue link.

Reply:
Good catch, thanks, the test was incorrect. Updated this test and the subsequent tests.